services/migration_service/api/admin.py

```diff
@@ -69,7 +69,7 @@ async def upgrade(self, request):
                 description: could not upgrade
         """
         goose_version_cmd = goose_migration_template.format(
-            database_name, user, password, host, port,
+            db_conf.connection_string,
             "up"
```

Comment:
It would still be a good idea to do `shlex.join` (or Py3.7 equivalent) here instead of `.format`

Reply:
Done.

Reply:
Done